lib/gamnit/tileset.nit

```diff
@@ -83,6 +83,12 @@ class TileSetFont
 	# A negative value may display overlapped tiles.
 	var vspace: Numeric = 0.0 is writable
 
+	# Line spacing modifier for `pld` and `plu`
+	#
+	# This value acts as multiplier to `height + vspace`.
+	# Defaults to 0.4, so a `pld` moves chars down by about half a line.
+	var partial_line_mod: Numeric = 0.4 is writable
```

Comment:
This is probably not the kind of screenshot that you were expecting, but here it is anyway. I use `pld` for 1.4 line skips under the titles and between wrapped lines.

![2017-05-09 08 54 31](https://cloud.githubusercontent.com/assets/208057/25852262/c5693448-3496-11e7-877b-da820790e1ca.jpg)